src/org/zaproxy/zap/extension/ascanrules/TestSQLInjection.java

```diff
@@ -75,7 +73,14 @@
 	 * Prefix for internationalised messages used by this rule
 	 */
 	private static final String MESSAGE_PREFIX = "ascanrules.testsqlinjection.";
-	
+	/**
+	* Did SQLInjection get found yet?
+	*/
+	private boolean sqlInjectionFoundForUrl = false;
```

Comment:
The code looks a little nicer as other `false` initialisation were already there.